common/src/test/java/tachyon/security/authentication/AuthenticationFactoryTest.java

```diff
@@ -135,8 +140,20 @@ public void simpleAuthenticationTest() throws Exception {
     client.open();
     Assert.assertTrue(client.isOpen());
 
+    // check user 'anyone'
+    verifyClientUser(client, "anyone");
+
+    // another connection by user 'anyone1'
+    TTransport client1 = createClient(mTachyonConf, "anyone1", "whatever");
```

Comment:
I don't actually see the reason to check another username here, does it actually add more code coverage?